manifests/server/apache.pp

```diff
@@ -47,17 +25,34 @@
     purge_configs  => $purge_configs,
   }
 
+  # TODO Audit this list. Drupal actually doesn't need most of these.
```

Comment:
Got it. I've tried to find a comprehensive list of what is actually needed and ended up with a much smaller list but this seems okay to me. They had autoindex in that list. I wonder if that was for a reason.